src/pages/Checkout/CheckoutPage.vue

```diff
@@ -493,6 +498,21 @@ export default {
 		// TODO: Implement check against contentful setting
 		// to signify if holiday mode is enabled
 
+		// VUE-1725 Async Checkout Mutation Rollout
+		const asyncCheckoutExp = this.apollo.readFragment({
+			id: `Experiment:${ASYNC_CHECKOUT_EXP}`,
+			fragment: experimentVersionFragment,
+		}) || {};
+
+		this.asyncCheckoutActive = asyncCheckoutExp?.version === 'b';
+		if (asyncCheckoutExp?.version !== null) {
```

Comment:
Should this just be `if (asyncCheckoutExp?.version)`? I think the default "no assignment" is `undefined`

Reply:
I'm getting this for the version when there is no setting. Should I check against both null and undefined?
`asyncCheckoutExp Object { __typename: "Experiment", id: "async_checkout_rollout", version: null }`

Reply:
We've just been comparing against truthy. But I was wondering if the default from apollo would be `null` instead.